src/zrc/conf/base.py

```diff
@@ -362,3 +363,11 @@
             'dsn': RAVEN_CONFIG['dsn']
         },
     })
+
+
+# settings for sending notifications
+NOTIFICATIES_URL = 'http://127.0.0.1:8001/api/v1/notificaties'
+NOTIFICATIES_KANAAL = 'zaken'
+NOTIFICATIES_KANAAL_URL = 'http://127.0.0.1:8001/api/v1/kanaal'
```

Comment:
this should be read-out from the API spec, given the base URL

Reply:
fixed using `zds_client.Client` and `zds_client.get_operation_url`

Reply:
Perhaps it's the reason of build errors. Because now a real request to other app's schema is needed in the settings of the project

Reply:
replaced NC schema request from settings to explicit url for test passing